core/src/main/java/io/questdb/cairo/TableWriter.java

```diff
@@ -1556,6 +1681,133 @@ public void warmUp() {
         }
     }
 
+    private void checkDetachedMetadata(long timestamp) {
+        // load/check _meta
+        other2.of(detachedPath).concat(META_FILE_NAME).$();
+        if (!ff.exists(other2)) {
+            // Backups and older versions of detached partitions will not have _meta
+            LOG.info().$("detached _meta not found, skipping check [path=").$(other2).I$();
+            return;
+        }
+        try {
+            detachedMetadata.deferredInit(other2, ColumnType.VERSION);
```

Comment:
What if old partition metadata is of different VERSION. We should still be able to attach it. Let's discuss it

Reply:
this has in effect gone, no further reverence to ColumnType.VERSION